### DISCUSSION TITLE
https://413e54c315bbb412-dot-us-central1.notebooks.googleusercontent.com/lab/tree/generative-ai/language/orchestration/langchain/intro_langchain_palm_api.ipynb

### DISCUSSION THREAD

**Discussion by jagadish-ganaparthi** [GoogleCloudPlatform/generative-ai#437] [General]
Same code is not working from CloudFunction.It is not giving erros

**Comment by holtskinner** [comment]
This notebook is using the PaLM model, which will reach end of life in April 2025 https://cloud.google.com/vertex-ai/generative-ai/docs/deprecations/palm
